test/core/test/basic.test.ts

```diff
@@ -3,6 +3,13 @@ import { two } from '../src/submodule'
 import { timeout } from '../src/timeout'
 
 test('Math.sqrt()', async() => {
+  // eslint-disable-next-line no-console
+  // console.log(`\x1B[33m${new Date().toISOString()}\x1B[0m`)
+  // eslint-disable-next-line no-console
+  // console.log(`\x1B[31m${new Error('I\'m a red fg error').stack}\x1B[0m`)
+  // eslint-disable-next-line no-console
+  // console.log(`\x1B[37m\x1B[41m${new Error('I\'m a red bg error').stack}\x1B[0m`)
+  // console.error('console error 1', new Error('hey there 2'))
```

Comment:
As UI testing with Cypress is enable on Vitest, it could be great to cover your work with it :)

Reply:
I need to refactor the console view component a few to remove the client (birpc) entry (add a new component and use it on the component) and so we can only check the component, later I'll try to add a few cypress tests on ui package

Reply:
Let me ask help or @JessicaSachs who will for sure be happy to help you!

Reply:
@edimitchel check the console cy test : https://imgur.com/Tq4zB9g

Reply:
working on report tab tests

Reply:
@edimitchel check the report cy test: https://imgur.com/gcPyYmH